SharedContent/cpp-winui-template/WinUI3TemplateCpp/SettingsPage.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<Page
+    x:Class="WinUI3TemplateCpp.SettingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUI3TemplateCpp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid RowDefinitions="*,Auto">
+
+        <StackPanel HorizontalAlignment="Left" Spacing="20" >
+            <TextBlock  Style="{StaticResource TitleTextBlockStyle}"
+                                Margin="0,4,0,0"
+                                FontWeight="Normal"
+                                Text="Theme Mode" />
+            <StackPanel x:Name="themePanel">
+                <RadioButton Tag="Light" Checked="OnThemeRadioButtonChecked" Content="Light"  />
```

Comment:
Don't these radio buttons need a GroupName?

Reply:
According to https://docs.microsoft.com/windows/apps/design/controls/radio-button#create-your-own-radiobutton-group, it is implicitly grouped since they are in the same stack panel